index.html

```diff
@@ -1,8 +1,36 @@
-Annah Nyarangi
-About
-I am a beginner programmer, currently working on JavaScript. I look forward to learning more. Currently working as a nursing assistant and hoping to make a career switch from healthcare to Information Technology.
-Experience
-2015 - Current 
-Nursing Assistant, GI Acute Care
-Connect
-https://github.com/anyarangi
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>
+           Annah Nyarangi code the dream JavaScript project.
+        </title>
+        <meta charset="UTF-8">
+        <meta name="keywords" content="HTML, CSS, JavaScript">
+        <meta name="author" content="Annah Nyarangi">
+        <link rel="stylesheet" href="style.css">
+    </head>
+
+    <body>
+        <section> 
+                <h1> Annah Nyarangi</h1>
```

Comment:
Move the h1 element above the first section (before starting section opening tag)